src/packages/cli/src/cli.ts

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env node
 
-import Ganache from "../index";
+import Readline from "readline";
+import Ganache, { Status } from "../index";
```

Comment:
I feel like this should be

```suggestion
import Ganache, { Status } from "@ganache/core";
```

`cli/index.ts` doesn't do any magic, and there's no precedent to import already exported versions of a dependency